.pryrc

```diff
@@ -0,0 +1,134 @@
+def create name='test card', content='some content', type='basic'
```

Comment:
As we're adding developers, we probably want to up our commenting game.  For the top section, we don't necessarily need a comment for each method, but maybe something like:
> These commands are available in the console when using `binding.pry` for breakpoints.

Reply:
the file name is `pryrc` and whenever you enter a breakpoint you get a short description of all commands (the puts command at the bottom of this file) but of course an extra comment does't hurt.

Reply:
I was thinking to make our breakpoint patterns (ie, the simple fact that pry is for breakpoints) more discoverable for new monkeys/platypi.